test/integration/snapshotcloneserver/snapshotcloneserver_concurrent_test.cpp

```diff
@@ -530,7 +530,7 @@ TEST_F(SnapshotCloneServerTest, TestCancelAndMakeSnaphotConcurrent) {
                     CancelSnapshot(testUser1_, testFile1_, uuid1);
                     isCancel = true;
                 }
-                std::this_thread::sleep_for(std::chrono::milliseconds(3000));
+                std::this_thread::sleep_for(std::chrono::seconds(30));
```

Comment:
sleep 3 seconds will not make concurrent ? Will 30 seconds make an affect to unit test in CI ？

Reply:
> sleep 3 seconds will not make concurrent ? Will 30 seconds make an affect to unit test in CI ？

Because we set task to cancel status in asynchronous model now, so maybe 3 seconds can't let the handler thread to capture it.